conda-forge.yml

```diff
@@ -1,5 +1,6 @@
 build_platform:
   linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
This enables cross-compilation. This can be a useful option since it allows building on `x86_64`, which works on Azure. This has a similar build time to building natively, which is significantly faster than an emulated build (what was being used before).

This was already enabled for `linux_aarch64`, but wasn't enabled for `linux_ppc64le`. Doesn't appear there was an issue with it from exploring the commit history. So giving it a try here to see if we can cutdown the build time for these CI jobs.